src/dgipy/vcf.py

```diff
@@ -0,0 +1,202 @@
+"""Provides methods for annotating VCF with DGIdb data"""
+
+import contextlib
+from itertools import groupby
+
+import pandas as pd
+import pysam
+import requests
+from tqdm import tqdm
+
+import dgipy
+
+# Sample usage: import vcf
+#               data = vcf.annotate('link/to/file',chr='chr#')
```

Comment:
This can go in the module docstring. Use the sphinx code-block directive (https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block) for example code

Reply:
done